server/email_verification.go

```diff
@@ -175,7 +200,7 @@ func handleVerifyEmailResendFunc(
 			}
 		}
 
-		_, err = emailer.SendEmailVerification(usr.ID, clientID, *redirectURL)
+		_, err = emailer.SendEmailVerification(usr.ID, clientID, *redirectURL, state, nonce, scopes)
```

Comment:
I don't think this is the right place to grab these value from. Sorry am trying to track the email sending code.